index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./style.css">
+	<title>Original Trombone</title>
+</head>
+<body>
+	<header>
+		<img src="./img/original_trombones.png" alt="Original Trombones">
```

Comment:
I would recommend that you wrap the page logo in an `<a>` tag as this would aid user experience. Your users should be able to navigate to the homepage by clicking on the page logo. Kindly check this 😁

Reply:
I think it would be best if your comment sounds friendly and not like orders as pointed out in the [guidelines](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

Remember to be polite and clear in your comments - our job here is to help another person improve in a friendly way. 
> You could say something like this: _I think it would be better to wrap the page logo in an `<a>` tag. This would aid user experience and user interaction in your app_

Reply:
I would suggest you point out the line you are referring to and be clear in your comment. 

You could say something like this; **The `<img>` tag which represents the logo needs to be wrapped in an `<a>` tag, this would aid user experience and user interaction.**